build/BuildScript.cs

```diff
@@ -84,40 +70,47 @@ protected override void ConfigureTargets(ITaskContext context)
                         .OutputDirectory(OutputDir));
                 });
 
-
-            var branch = context.BuildSystems().Travis().BranchName;
-
-            var packs = context.GetFiles(OutputDir,"*.*").Where(x => x.FileName.Contains(Version));  
```

Comment:
packs are always empty as GetFiles is executed in ConfigureTargets which gets executed by flubu runner before target dependecnies and tasks that were added to target therefore output directorz at the time GetFiles is executed is still empty. To solve this i've moved the code to the Do method in this case code in Do method will be executed when push target is executed.

You can read more about this in faq